tests/test_core.py

```diff
@@ -1304,13 +1305,19 @@ async def coroutine_listener(event):
 async def test_eventbus_max_length_exceeded(hass: HomeAssistant) -> None:
     """Test that an exception is raised when the max character length is exceeded."""
 
+    await async_setup_component(hass, "homeassistant", {})
+
```

Comment:
Probably should have a test for if `homeassistant ` isn't loaded yet

Reply:
Test case added